Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM  node:alpine as builder
+
+# needed to build (python needed for compiling node-sass, for instance...)
+RUN apk add --no-cache \
+    make \
+    gcc \
+    g++ \
+    python
+
+WORKDIR /opt/pizza-sync
+ADD . .
+
+# frontend
+RUN cd frontend && yarn && yarn run build:prod && rm -rf node_modules
+
+# backend
+RUN cd backend/ && yarn
+
+# insert compiled front in backend dir
+RUN mv frontend/dist backend/public
```

Comment:
I think we can merge front and back build and then we should move the copy into the last stage (in one run also).
But it's not so important because it is the build stage, so it won't be uploaded. It's more important to ensure there is a minimum of run in the last stage.

By the way, note that nowadays docker can squash all the layers into one, so it could also be a way to handle that situation (I think it is an option to pass to docker build).